### PR TITLE
feat/code-quality-improvements

### PR DESCRIPTION
**Pull request by AndriiBogomolov**
Please check out performed task. All is done according to the task - FE Project Setup and Code Quality Improvements.

### CODE REVIEW

File: frontend/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": true,
+  "bracketSpacing": true,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "printWidth": 80,
```
Comment (whateverfw) [inline frontend/.prettierrc:8]: I'd change printWidth to at least 100 or even 120

File: frontend/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": true,
```
Comment (whateverfw) [inline frontend/.prettierrc:4]: it is better to keep double quote for jsx specifically

File: frontend/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+//       "jsx": true
+//     }
+//   },
+//   "settings": {
+//     "react": {
+//       "version": "detect"
+//     }
+//   }
+// }
+{
```
Comment (whateverfw) [inline frontend/.eslintrc.json:62]: why is this commented out?

File: frontend/package.json [hunk 4]
```diff
@@ -10,27 +10,19 @@
...
+    "react-router-dom": "^6.27.0",
     "react-scripts": "^5.0.1",
+    "react-toastify": "^10.0.6",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "jest",
+    "lint": "eslint .",
```
Comment (whateverfw) [inline frontend/package.json:22]: also add separate command for prettier, so in the future both eslint and prettier can be run in the CI

File: frontend/src/components/LendingForm.js [hunk 5]
```diff
@@ -1,180 +1,201 @@
...
+  const fetchBalances = useCallback(async () => {
+    if (!walletId) {
+      return;
+    }
+    try {
+      const tokenBalances = await getTokenBalances(walletId);
+      setBalances(tokenBalances);
+    } catch (error) {
+      console.error('Failed to fetch balances:', error);
+      alert('Failed to fetch token balances. Please try again.');
```
Comment (whateverfw) [inline frontend/src/components/LendingForm.js:26]: change to notify

File: package.json [hunk 6]
```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
     "starknetkit": "^2.3.0"
+  },
+  "devDependencies": {
+    "eslint-plugin-import": "^2.29.1"
```
Comment (whateverfw) [inline package.json:6]: it seems like it was installed here accidentally?

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/.prettierrc (hunk 1 above)
Comment (AndriiBogomolov) [reply frontend/.prettierrc]: Fixed. Please check it out. Thanks.

File: frontend/.prettierrc (hunk 2 above)
Comment (AndriiBogomolov) [reply frontend/.prettierrc]: Fixed. Please check it out. Thanks.

File: frontend/.eslintrc.json (hunk 3 above)
Comment (AndriiBogomolov) [reply frontend/.eslintrc.json]: Sorry, the commented part is deleted now. Thank you.

File: frontend/package.json (hunk 4 above)
Comment (AndriiBogomolov) [reply frontend/package.json:22]: Prettier script was added. Please check it out. Thanks.

File: frontend/src/components/LendingForm.js (hunk 5 above)
Comment (AndriiBogomolov) [reply frontend/src/components/LendingForm.js]: It is changed to notify. Please check it out. Thanks.

File: frontend/.eslintrc.json
Comment (whateverfw) [inline frontend/.eslintrc.json:1]: seems like there are a couple of plugins installed, such as:
"eslint-config-prettier"
"eslint-plugin-react"
"eslint-plugin-react-hooks

but it is being used here

(e.g. you can take a look into examples of https://www.npmjs.com/package/eslint-plugin-react)

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/.eslintrc.json
Comment (AndriiBogomolov) [reply frontend/.eslintrc.json:1]: yes, these plugins are unnecessary here

Comment (whateverfw) [reply frontend/.eslintrc.json:1]: I meant they are missing here and should be added

Comment (AndriiBogomolov) [reply frontend/.eslintrc.json:1]: Please see added ESLint plugins. Please check it out and merge if all is right. I'm working on resolving conflict as well.

File: frontend/src/App.jsx [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
 import { Routes, Route, Navigate, useNavigate } from 'react-router-dom';
-import Header from './components/header/Header';
-import Dashboard from './pages/spotnet/dashboard/Dashboard';
-import Footer from './components/Footer/Footer';
-import SpotnetApp from './pages/spotnet/spotnet_app/SpotnetApp';
-import Login from "./pages/Login";
-import Form from "./pages/forms/Form";
-import { connectWallet, logout } from './utils/wallet';
-import { saveTelegramUser, getTelegramUserWalletId} from './utils/telegram';
+import Header from 'components/Header/Header';
```
Comment (whateverfw) [inline frontend/src/App.jsx:3]: This should be `components/header/Header` with `header` starting from a lowercase letter, currently this line seems to cause build error and failed CI

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**